### ISSUE TITLE
[Web] CardField #card-element selector applies to no DOM elements

### ISSUE THREAD

**Issue by BakerSmithA** [flutter-stripe/flutter_stripe#1925]
**Describe the bug**
Looks to be a duplicate of a previously fixed issue https://github.com/flutter-stripe/flutter_stripe/issues/1680 that happens when a `CardField` widget is wrapped in an `AnimatedOpacity` widget that starts with zero opacity. After animating from zero to one opacity, the CardField widget still does not display or allow input, and the following error is printed to the terminal:

```
IntegrationError: The selector you specified (#card-element) applies to no DOM elements that are currently on the page.
Make sure the element exists on the page before calling mount().
```

(I assume the Flutter engine is doing some optimisations and not initalising the `CardField`).

**To Reproduce**
See this repo which provides an example:

https://github.com/BakerSmithA/flutter_stripe_opacity_example

**Expected behavior**
The CardField widget allows card detail input and builds without error.

**Smartphone / tablet**
 - Device: Macbook Pro
 - OS: Sonoma
 - Package version: flutter_stripe 6.0.0, flutter_stripe_web 11.0.0
 - Flutter version 3.24.2

**Additional context**
This issue occurs after upgrading from flutter_stripe version [5.2.0](https://pub.dev/packages/flutter_stripe_web/versions/5.2.0) to [6.0.0](https://pub.dev/packages/flutter_stripe_web/versions/6.0.0), and flutter_stripe_web from [10.2.0](https://pub.dev/packages/flutter_stripe/versions/10.2.0) to [11.0.0](https://pub.dev/packages/flutter_stripe/versions/11.0.0).

**Comment by remonh87** [comment]
@jamesblasco  I think we can change this widget using the same mutation observer trick that we use for paymentelement am I correct?

**Comment by remonh87** [comment]
Closed by #1978

**Comment by aiiro-s** [comment]
Testing to get comments
